docs/git.rst

```diff
@@ -0,0 +1,304 @@
+.. -*- coding: utf-8 -*-
+
+==================
+git best practices
+==================
+Collection of best practices that would help you and the overall community to work with git more efficiently.
+
+General guidelines
+==================
+
+Pulling code
+------------
+*ALWAYS* (in capital letters) do a :command:`git pull -r` when fetching new code,
```

Comment:
Good point @jean 

Actually as I got the idea that it was a bit to much into the point, right now I'm writing a little introduction, hopefully it doesn't end up being a duplicate of git's own book: http://git-scm.com/book/en/v2 :)